sdk/core/Azure.Core.Expressions.DataFactory/tests/DataFactoryExpressionsTests.cs

```diff
@@ -259,10 +274,27 @@ private static void AssertBoolDfe(DataFactoryExpression<bool> dfe)
         public void DeserailizationOfStringValue()
         {
             var doc = JsonDocument.Parse(StringJson);
-            var dfe = DataFactoryExpression<string>.DeserializeDataFactoryExpression(doc.RootElement);
+            var dfe = DataFactoryExpressionJsonConverter.Deserialize<string>(doc.RootElement);
             AssertStringDfe(dfe, StringValue);
         }
 
+        [Test]
+        public void DeserializationOfListOfT()
```

Comment:
Can we add a case to validate the case where we call DataFactoryExpression<List<TestModel>>.FromExpression("blah")?

I assume the json should either be the literal array OR the normal expression object?  Should we validate this with the service team?

Reply:
Checked with service team - the expression object would have the same structure. Will add a test.